GitUI/CommandsDialogs/SettingsDialog/Pages/GeneralSettingsPage.cs

```diff
@@ -156,5 +138,10 @@ private void ShowGitStatus_CheckedChanged(object sender, System.EventArgs e)
         {
             SetSubmoduleStatus();
         }
+
+        private void LlblTelemetryPrivacyLink_LinkClicked(object sender, System.Windows.Forms.LinkLabelLinkClickedEventArgs e)
+        {
+            Process.Start("https://github.com/gitextensions/gitextensions/blob/master/PrivacyPolicy.md");
```

Comment:
This links to master. Should we link to the release branch instead? Or the wiki?

Reply:
Good point. I think linking to the branch's doc is better because it outlines what each version captures.
In set_version_to.py we update the link to the docs, in the same manner we can update the link to the branch's document:
https://github.com/gitextensions/gitextensions/blob/5e825a186d8ea793be4c3bff9893ab56b158700b/Setup/set_version_to.py#L38-L45

Reply:
Mulling it over, the wiki points to [the master version for COC](https://github.com/gitextensions/gitextensions/blob/master/CODE_OF_CONDUCT.md). 
So it is probably ok to point to the privacy policy in the same fashion.

We can create a separate wiki page, if you think it is worth it, but I don't see much value in it.

I have also played with the py script, but it seems complex and brittle (with my limited knowledge of python).